src/Microsoft.ML.Transforms/UngroupTransform.cs

```diff
@@ -359,14 +389,19 @@ public PivotColumnInfo GetPivotColumnInfo(int iinfo)
 
             public PivotColumnInfo GetPivotColumnInfoByCol(int col)
             {
-                _ectx.Assert(0 <= col && col < ColumnCount);
+                _ectx.Assert(0 <= col && col < OutputSchema.Count);
```

Comment:
>OutputSchema.Count [](start = 47, length = 18)

same here

Reply:
Fixed.

---
In reply to: [246128188](https://github.com/dotnet/machinelearning/pull/2066#discussion_r246128188) [](ancestors = 246128188)